src/main/resources/org/joda/money/MoneyData.csv

```diff
@@ -23,6 +23,7 @@ BRL,986,2,BR
 BSD,44,2,BS
 BTN,64,2,BT
 BWP,72,2,BW
+BYN,-1,2,BY
```

Comment:
should be `BYN,933,2,BY`
You also have to comment out `BYR` or the data won't load